[2022-09-27 17:13:22,191] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-09-27T17:13:14.714798+00:00 [queued]>
[2022-09-27 17:13:22,205] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-09-27T17:13:14.714798+00:00 [queued]>
[2022-09-27 17:13:22,205] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-27 17:13:22,206] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-27 17:13:22,206] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-27 17:13:22,221] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-09-27 17:13:14.714798+00:00
[2022-09-27 17:13:22,232] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'manual__2022-09-27T17:13:14.714798+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpma7cz09m', '--error-file', '/tmp/tmpjzr1e517']
[2022-09-27 17:13:22,229] {standard_task_runner.py:52} INFO - Started process 116 to run task
[2022-09-27 17:13:22,234] {standard_task_runner.py:77} INFO - Job 138: Subtask download_dataset_task
[2022-09-27 17:13:22,327] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-09-27T17:13:14.714798+00:00 [running]> on host f5057273dea0
[2022-09-27 17:13:22,409] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-09-27T17:13:14.714798+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-27T17:13:14.714798+00:00
[2022-09-27 17:13:22,410] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-27 17:13:22,413] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://people.sc.fsu.edu/~jburkardt/data/csv/crash_catalonia.csv > /opt/***//crash_catalonia.csv']
[2022-09-27 17:13:22,452] {subprocess.py:85} INFO - Output:
[2022-09-27 17:13:26,250] {subprocess.py:93} INFO - Command exited with return code 0
[2022-09-27 17:13:26,342] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20220927T171314, start_date=20220927T171322, end_date=20220927T171326
[2022-09-27 17:13:26,400] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-27 17:13:26,557] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
